docs/csharp/programming-guide/inside-a-program/identifier-names.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Identifier names"
+description: "Learn the rules for valid identifier names in the C# programming language."
+ms.date: 08/21/2018
+---
+# Identifier names
+
+An **identifier** is the name you assign to a type (class, interface, struct, delegate, or event), member, variable, or namespace. Valid identifiers must follow these rules:
```

Comment:
>type (class, interface, struct, delegate, or event)

The [Members](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/members) article refers to events as members, while enum is not mentioned in the list of possible types. So, should it be:
>type (class, interface, struct, delegate, or enum)